onnxruntime/core/graph/contrib_ops/reverse_sequence_schema_defs.cc

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "reverse_sequence_schema_defs.h"
+#include "core/graph/constants.h"
+#include "core/graph/op.h"
+#include <cmath>
+#include <type_traits>
+
+namespace onnxruntime {
+namespace contrib {
+
+using ::ONNX_NAMESPACE::AttributeProto;
```

Comment:
>:: [](start = 6, length = 2)

nit: pet peeve. unnecessary usage of '::' as a prefix. We should only ever have one ONNX_NAMESPACE, so the global '::' qualifier as a prefix should never be necessary. I know this was copied from existing defs though.